api/src/index.ts

```diff
@@ -29,9 +32,78 @@ async function startServer() {
 	const version = CONFIG.apiVersion;
 	const loggerMiddleware = morgan('combined');
 
+	const options = {
```

Comment:
please move options somewhere else and import them maybe routers or docs is a folder for that